web/src/user_group_popover.js

```diff
@@ -49,40 +50,57 @@ export function handle_keyboard(key) {
 // element is the target element to pop off of;
 // message_id is the message id containing it, which should be selected;
 export function toggle_user_group_info_popover(element, message_id) {
-    const $last_popover_elem = $current_user_group_popover_elem;
-    hide_all();
```

Comment:
So the `hide_all` here closed existing unrelated popovers when opening this one. 

I think that was desired behavior, and notice that most of our `popover_menus` popovers use either `on_show_prep` or `popovers.hide_all_except_sidebars`. I'll add the latter but it's probably worth doing an audit for whether any of the popovers we converted are doing this wrong.